templates/mas_config.yaml.j2

```diff
@@ -0,0 +1,55 @@
+http:
+  listeners:
+  - name: web
+    resources:
+    - name: discovery
+    - name: human
+    - name: oauth
+    - name: compat
+    - name: graphql
+    - name: assets
+    binds:
+    - address: '[::]:8081'
+    prefix: /auth/
+    proxy_protocol: false
+  - name: internal
+    resources:
+    - name: health
+    binds:
+    - host: localhost
+      port: 8082
+    proxy_protocol: false
+  trusted_proxies:
+  - 192.168.0.0/16
```

Comment:
Reminder: how does mas work with proxy?

Reply:
I removed this for now and will follow-up with matrix team on what `trusted_proxies` means